### DISCUSSION TITLE
How to debug Console not loading all modules?

### DISCUSSION THREAD

**Discussion by jelling** [blitz-js/blitz#2745] [Q&A]
Is there a process for debugging why `blitz console` isn't loading all modules? 

```
> blitz console

You have entered the Blitz console
Tips: - Exit by typing .exit or pressing Ctrl-D
      - Use your db like this: await db.project.findMany()
      - Use your queries/mutations like this: await getProjects({})
Loading Modules 20/22
⚡️ > 
```

I've had this problem on multiple occasions and the cause is typically because of an issue that's also visible in `npm run build` or `npm run dev`. But today it's not and I've tried everything including blowing away the local database, removing the `.blitz` and `.next` directories, and rebooting.

Is there some way to run this in verbose mode? Launching `npm run console` in vscode debug does not show any issues.

**Comment by jelling** [comment]
I solved my problem. The issue was not with loading modules; they were all loading correctly, which I verified by debugging `repl.ts`. The real problem was that a mutation was not showing up in the console and I misinterpreted that as the modules not loading. 

The mutation in question was not being loaded because it referenced an environment variable that was missing. This meant the code would compile via `npm run build` but when console loads mutations it apparently evaluates the code in a run-time fashion even prior to the mutation. Unfortunately, though, if it fails due a missing ENV var it does so silently.

**Reply by beerose** [reply]
Thanks for the reply! I created an issue to track the silent fail: https://github.com/blitz-js/blitz/issues/2751
